tools/create-trials-index.js

```diff
@@ -358,13 +359,25 @@ function indexAutocompleteModel(model, indexType) {
     model,
     'autocomplete',
     indexType,
-    { columns: ['id', 'name'], withRelated: 'trials' }
+    {
+      // Filter out entities without trials
+      innerJoin: [
```

Comment:
Would be interesting to see if this approach, while consuming more resources for the join, will still be under the footprint Node currently has. Looking forward! :)